src/rules/comment-word-blacklist/__test__/index.js

```diff
@@ -0,0 +1,195 @@
+import testRule from "../../../testUtils/testRule"
+import rule, { ruleName, messages } from ".."
+
+testRule(rule, {
+  ruleName,
+  config: [[
+    "/^TODO:/",
+    "bad-word",
+  ]],
+
+  accept: [ {
+    code: "/* comment */",
+  }, {
+    code: "/* comment comment */",
+  }, {
+    code: "/* comment\ncomment */",
+  }, {
+    code: "/* comment\n\ncomment */",
+  }, {
+    code: "/** comment */",
+  }, {
+    code: "/**** comment ***/",
+  }, {
+    code: "/*\ncomment\n*/",
+  }, {
+    code: "/*\tcomment   */",
+  }, {
+    code: "/*! copyright */",
+  }, {
+    code: "/*# sourcemap */",
+  }, {
+    code: "a { color: pink; /* comment */\ntop: 0; }",
+  }, {
+    code: "a {} /* comment */",
+  }, {
+    code: "/* todo */",
+  }, {
+    code: "/* todo: */",
+  }, {
+    code: "/* todo: comment */",
+  }, {
+    code: "/* tOdO: comment */",
+  }, {
+    code: "/* Todo: comment */",
+  }, {
+    code: "/* Comment with bad-word */",
+  }, {
+    code: "/*! Todo: comment */",
+  }, {
+    code: "/*# Todo: comment */",
+  }, {
+    code: "/*! bad-word */",
+  }, {
+    code: "/*# bad-word */",
+  }, {
+    code: "/** bad-word **/",
+  }, {
+    code: "/*** bad-word ***/",
+  }, {
+    code: "/** TODO: comment **/",
+  }, {
+    code: "/*** TODO: comment ***/",
+  } ],
+
+  reject: [ {
+    code: "/* TODO: */",
+    message: messages.rejected("TODO:"),
+    line: 1,
+    column: 1,
+  }, {
+    code: "/* TODO: comment */",
+    message: messages.rejected("TODO: comment"),
+    line: 1,
+    column: 1,
+  }, {
+    code: "/* TODO: comment\n next line */",
+    message: messages.rejected("TODO: comment\n next line"),
+    line: 1,
+    column: 1,
+  }, {
+    code: "/* TODO: comment\n next line */",
+    message: messages.rejected("TODO: comment\n next line"),
+    line: 1,
+    column: 1,
+  }, {
+    code: "/* TODO: comment\r\n next line */",
+    message: messages.rejected("TODO: comment\r\n next line"),
```

Comment:
Perhaps there's a nicer solution here than displaying the whole comment in the message? If a user defined regex returns *more than* a word i.e. there are spaces or newlines.. then set the rejected message to an empty string so that the final message will just be "Unexpected word (comment-word-blacklist)" in those instances?

Is that the best compromise?